src/index.js

```diff
@@ -164,20 +162,21 @@ const initInterface = () => {
     });
     tabsFilter = new Filter(opts, [config.tabFilters.defaultTab]);
     tabsFilter.set(config.tabFilters.defaultTab, true);
-    tabsFilter.listen(() => {
+    tabsFilter.listen(selectionUpdatedEvent, () => {
       // Perform a search when the tabs change
       if (!disableTabFacetSearch) {
-        values.emitChange();
-        pipeline.search(values, tracking);
+        values.emitUpdated();
+        pipeline.search(values.get());
       }
     });
 
-    values.listen(valuesChangedEvent, changes => {
+    values.listen(valuesUpdatedEvent, changes => {
       // If the query is empty, reset the tab back to the default if it's not already
       if (
         !values.get().q &&
         tabsFilter.get() !== config.tabFilters.defaultTab
       ) {
+        console.log();
         disableTabFacetSearch = true;
```

Comment:
not that I can remember, the selection/option distinction in the filter doesn't help this situation. I agree it's not great.